requirements.txt

```diff
@@ -2,3 +2,4 @@ six
 tqdm
 torchtext==0.1.1
 future
+pyrouge
```

Comment:
can you put this in requirements.opt.txt

https://github.com/OpenNMT/OpenNMT-py/blob/master/requirements.opt.txt